bin/extract-ORFs.py

```diff
@@ -39,20 +39,35 @@
 
     parser.add_argument(
         '--minORFLength', metavar='LEN', type=int, default=None,
-        help='Only ORFs of at least this length will be written to stdout.')
+        help=('Only ORFs of at least this length will be written to stdout.'))
```

Comment:
We only use the parens if the help text is longer than one line :-)

Reply:
ah yes :) done